api/templates/registration/password_change_form.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block body %}
+
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="{% url 'home' %}">Home</a></li>
+        <li class="breadcrumb-item">Change password</li>
+    </ol>
+</nav>
+
+  <h2>Change password</h2>
+  <form method="post">
```

Comment:
This doesn't have the bootstrap theme on the form.

Reply:
I'm sorry, but I don't understand. Wouldn't the bootstrap theme be included in the base.html, which is extended in this file?